packages/ontario-frontend-cli/create/templates/README.njk

```diff
@@ -4,7 +4,7 @@ Built with the [Ontario Frontend toolkit](https://github.com/ongov/ontario-front
 
 ## Project Description
 
-{{ projectDescription }}
+Ontario Frontend Project (Add your project description).
```

Comment:
Can we make this a bit more of a call-out?

For example:
`Ontario Frontend Project (TODO: Replace with a description of your project)`